### DISCUSSION TITLE
SK Office Hours

### DISCUSSION THREAD

**Discussion by rajyraman** [microsoft/semantic-kernel#8178] [General]
I generally watch only the recording due to me being in GMT + 11. The problem I have is that none of the recordings on YouTube have the links being talked about in the video. It would be helpful to have these in the video or in the discussion area after each Office Hours call. Even the [recent one](https://youtu.be/HDxcmlBYyas?si=eXn25fVUIaqEol-v) has no links.

**Comment by sophialagerkranspandey** [comment]
We'd recommend you look at the following links for the content shared:
[Semantic Kernel | The latest news from the Semantic Kernel team for developers (microsoft.com)](https://devblogs.microsoft.com/semantic-kernel/)
[Backlog by Bucket · Semantic Kernel (github.com)](https://github.com/orgs/microsoft/projects/866/views/59)
[Introduction to Semantic Kernel | Microsoft Learn](https://learn.microsoft.com/en-us/semantic-kernel/overview/)
